xtask/src/vm.rs

```diff
@@ -15,6 +15,7 @@
 //
 
 const CLIENT_PATH: &str = "./target/debug/oak_functions_client";
+const WASM_PATH: &str = "./experimental/oak_baremetal_loader/key_value_lookup.wasm";
```

Comment:
Probably because I am missing a lot of context + pre-existing, but: why do we hard-code the "key_value_lookup.wasm"? Why not, e.g., "echo.wasm"?

Is the redirection to the `WASM_PATH` variable worth it or could we inline?

Reply:
If it helps: Here I use [WORKSPACE_ROOT](https://github.com/project-oak/oak/blob/228236fefa5c8279b42138d2452b7fd87f7ff9b1/oak_functions/sdk/oak_functions/tests/integration_test.rs#L32)

Reply:
Mostly we do just that because this way we can test that the lookup logic works. :) I guess in the future we'd want to also have multiple examples for the bare-metal runtime, which test different functionality.

Reply:
Re [WORKSPACE_ROOT](https://github.com/project-oak/oak/blob/228236fefa5c8279b42138d2452b7fd87f7ff9b1/oak_functions/sdk/oak_functions/tests/integration_test.rs#L32) I like the clarity of it, but realistically we'd have to then update all relative paths in this file, not sure if it's worth it. :)